### PR TITLE
assessment answers submitted

### PR DESCRIPTION
**Pull request by sarabrentmccoy**

### CODE REVIEW

File: lib/assessment.js [hunk 1]
```diff
@@ -8,15 +8,24 @@
 // Question 1
 // Write a callback for reduce that selects
 // the minimum value in the array
-var min = function min() {
 
+//no...
+var ary = [];
+var min = ary.reduce(function min(a, b) {
+  return
```
Comment (MattB-GA-Boston) [inline lib/assessment.js]: The question asks for a callback function, `min`, which we can pass into `reduce`, i.e. `[1,2,3].reduce(min);`

Here's an example of one we might have used.
```javascript
var min = function(previous, current) {
  if (previous < current) {
    return previous;
  } else {
    return current;
  }
}
```

File: lib/assessment.js [hunk 2]
```diff
@@ -8,15 +8,24 @@
...
 // the sum of the values in the array
-var sum = function sum() {
 
+//no...
+var numbers = [];
+var start = 0;
+numbers.forEach(
+  function sum(value) {sum += value;});
+
+console.log(sum);
```
Comment (MattB-GA-Boston) [inline lib/assessment.js]: See comment above.

**Comment by MattB-GA-Boston** [comment]
Let me know if you want to retake this - happy to give you feedback again. Alternatively, we could walk through Antony's solution branch together.

**Comment by sarabrentmccoy** [comment]
Thanks, Matt. I'm interested in retaking it, for sure -- I feel like I've
figured this out more-or-less since I first took the assessment.



On Mon, Oct 12, 2015 at 12:12 PM, Matt Brendzel <notifications@github.com>
wrote:

> Closed #7
> <https://github.com/ga-wdi-boston/js-array-methods-assessment/pull/7>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ga-wdi-boston/js-array-methods-assessment/pull/7#event-433111132>
> .
>



-- 
Sarabrent McCoy | m: (225) 773-2988

**Pull request closed without merging**